recipes/aiida-cusp/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "aiida-cusp" %}
+{% set version = "0.1.0b2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fc62a4474503eb12e296f11986c840e0b29ecf4a5ac17acbddd8efbaee3b3a78
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [win or py < 36]
```

Comment:
This can be noarch: python with `python >=3.6` below

Reply:
Maybe I missed something here, but wouldn't that also make the package installable on ``win`` platforms? (It's definitely not compatible!)

Reply:
We generally don't worry too much about it being possible to install an package that is unsupported on Windows and there are benefits to being `noarch: python` (single "binary", instant support for ARM/POWER, no need to rebuild for new Python versions).

In this case though it won't be possible to install as `aiida-core >=1.3.0,<2.0.0` isn't available for Windows.

Reply:
Oh well, if you do not care about packages being installable on incompatible platforms then this can indeed be turned into ``noarch``. Cool, thanks for the hint!